### ISSUE TITLE
No button to close fullscreen video call

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#19181]
### Steps to reproduce

1. Video call (native 1:1)
2. click the fullscreen button in the corner

### What did you expect?
same button in same corner to close fullscreen

### What happened?
no button to close fullscreen

### Related
#965

### Operating system

arch

### Application version

Element Nightly version: 2021092401 Olm version: 3.2.3

### How did you install the app?

aur nightly-bin

### Homeserver

private

### Have you submitted a rageshake?

No

**Comment by SimonBrandner** [comment]
I think we should move the minimize/maximize button into the call view. @gaelledel, do you have any thoughts on this?

**Comment by HarHarLinks** [comment]
I would agree. Personally I expected the maximize/minimize button to be with the other call controlling buttons when I looked for it. However the corner is also a familiar place from browser videos.
Iirc double-clicking the "video" to toggle fullscreen as we are used to from popular video streaming sites worked either.

**Comment by SimonBrandner** [comment]
I am for the top-right corner as it somehow feels the most natural to me

**Comment by HarHarLinks** [comment]
Further, desktop (do browsers do that already?) should maybe show a "press ESC to exit fullscreen" message for 3 seconds after fullscreening or so.

**Comment by Palid** [comment]
Bumping it up to s-major, as it seems there's no good workaround for it.

**Comment by SimonBrandner** [comment]
> Bumping it up to s-major, as it seems there's no good workaround for it.

Pressing escape seems ok to me

**Comment by dbkr** [comment]
We'll want to move this anyway if we take calls out of the widget view, so putting it with the other call buttons could make sense.

**Comment by SimonBrandner** [comment]
> We'll want to move this anyway if we take calls out of the widget view, so putting it with the other call buttons could make sense.

I would put it inside the call view but it feels weird in the bottom toolbar
